lib/generators/react_on_rails/templates/redux/base/client/app/lib/middlewares/loggerMiddleware.js

```diff
@@ -12,7 +12,7 @@ export default function logger({ getState }) {
 
     // We can't console.log immutable objects out-of-the-box.
     const immutableState = getState();
-    const readableState = _.reduce(immutableState, (result, immutable, key) => {
+    const readableState = _.reduce(immutableState, (object, immutable, key) => {
       result[key] = immutable.toJS();
```

Comment:
@justin808 Yep, you're right. I think the fact that even with the variable names sorted out, the code was just wrong, it's fixed now.